src/cryptsetup/cryptsetup-generator.c

```diff
@@ -99,11 +99,13 @@ static int split_keyspec(const char *keyspec, char **ret_keyfile, char **ret_key
         return 0;
 }
 
-static int generate_keydev_mount(
+static int generate_dev_mount(
                 const char *name,
-                const char *keydev,
-                const char *keydev_timeout,
+                const char *dev,
```

Comment:
maybe spell this out, actually, we don't like needless abbreviations, and since this doesn't directly mirror a user-facing concept anymore I think we should better spell this out as "device"

Reply:
Fixed